src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/restapi/config/SwaggerConfig.kt

```diff
@@ -28,4 +28,26 @@ class SwaggerConfig {
       .addOpenApiCustomizer(apiStatisticsOpenApiCustomizer())
       .build()
   }
+
+  @Bean
```

Comment:
I presume this will result in another drop down item in the swagger page? if so I'd rather just add these to the group above and tweak the title/ description